src/Form/Type/Operator/ContainsOperatorType.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Form\Type\Operator;
+
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType as FormChoiceType;
+use Symfony\Component\Form\FormTypeInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class ContainsOperatorType extends FormChoiceType
+{
+    public const TYPE_CONTAINS = 1;
```

Comment:
Can you make these private?

Reply:
I don't think, 

Since I deprecated the FilterType const
They will be used in the `Sonata\DoctrineORMAdminBundle\Filter`

```
ChoiceType::TYPE_CONTAINS => 'LIKE',
ChoiceType::TYPE_NOT_CONTAINS => 'NOT LIKE',
ChoiceType::TYPE_EQUAL => '=',
```
Or
```
NumberType::TYPE_EQUAL => '=',
NumberType::TYPE_GREATER_EQUAL => '>=',
NumberType::TYPE_GREATER_THAN => '>',
NumberType::TYPE_LESS_EQUAL => '<=',
NumberType::TYPE_LESS_THAN => '<',
```

Will be replace by
```
ContainsOperatorType::TYPE_CONTAINS => 'LIKE',
ContainsOperatorType::TYPE_NOT_CONTAINS => 'NOT LIKE',
ContainsOperatorType::TYPE_EQUAL => '=',
```

The filter need to know the operator value to write the right query.